api-matlab/rtQC_display.m

```diff
@@ -1456,6 +1464,36 @@ function generateReport(hObject,eventdata)
 
 end
 
+function physioBatch(hObject,eventdata)
+    % check if physio is installed:
+    pathCell = regexp(path, pathsep, 'split');
+    if ispc  % Windows is not case-sensitive
+      onPath = any(contains(pathCell, 'PhysIO'));
+    else
+      onPath = any(contains(pathCell, 'PhysIO'));
+    end
```

Comment:
The outcome of both `if` and `else` components will be identical, right? I see no difference in lines 1471 and 1473.